hub/examples/image_retraining/label_image.py

```diff
@@ -69,6 +69,13 @@ def wiki(celestial_object):
         statistics = "\n".join(['-- {}: {}'.format(parameter, value) for parameter, value in req_statistics.items()])
         print("{}\n\n".format(statistics))
         print(wikipedia.WikipediaPage(title='{}'.format(ans)).summary)
+    elif ans == 'asteroid':
```

Comment:
@Zignake This will be asteroids

```suggestion
    elif ans == 'asteroids':
```